### ISSUE TITLE
Protocal Name replaced by sequence name in some filenames

### ISSUE THREAD

**Issue by chrisfoulon** [rordenlab/dcm2niix#403]
Dear dcm2niix team, 

Thanks for your awesome work! 
I just noticed a bug in dcm2niiX version v1.0.20200331 when using the -f option with %p, the protocol name if replaced by the sequence name. 
And weirdly, it is not on every conversion, but for now, I cannot find a clear pattern.
I just upgraded from dcm2niiX version v1.0.20190902 and I didn't have this problem before. 

Thank you in advance for your help!
Chris.

**Comment by neurolabusc** [comment]
I believe the documentation needs to be upgraded to describe the current behavior. The text `%p` in a filename will be replaced by the DICOM ProtocolName ([0018,1030](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,1030))). However, if the ProtocolName is empty, or if the Manufacturer ([0008,0070](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0008,0070))) is `GE`, then the SequenceName ([0018,0024](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0024))) is used. The rationale is that GE uses a generic ProtocolName, with the SequenceName being equivalent to the SequenceName reported by Siemens and Philips.

I think this explains the behavior you are seeing, right?

**Comment by chrisfoulon** [comment]
Thank you for your quick answer.
Unfortunately no. I tried another image that was wrongly labelled and this time the ProtocalName was replaced by the StudyInstanceUID. 
Moreover, I am using pydicom to extract the dicom header information and try to match it with the output of dcm2niix, and every time I saw the ProtocalName replaced, pydicom could find it in the header, so I don't think this is normal behaviour. 

And if dcm2niix is indeed replacing missing attributes by something else, yes it would be very useful to have the table of replacement attributes :) 

Thank you.
